packages/stellar_solana_bridge/src/components/BridgeSolana.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-container class="d-flex justify-center">
+    <v-card class="pa-4" width="400">
+      <v-card-title class="text-center">TFT Transfer</v-card-title>
+
+      <!-- From Address -->
+      <v-text-field v-model="fromAddress" label="Stellar Account Seed" variant="outlined"></v-text-field>
+
+      <!-- To Address -->
+      <v-text-field v-model="toAddress" label="Solana Associated Token Address" variant="outlined"></v-text-field>
```

Comment:
add rules here to check address validity please if possible , you can also check for address length